frontend/src/components/Button/Button.tsx

```diff
@@ -1,3 +1,5 @@
+import "./Button.css";
```

Comment:
Ta cała klasa to jest typowy "component base styling". Z tego co widzę to tailwind jest raczej tego przeciwieństwem. @mejsiejdev Czy jest to normalną praktyką żeby w tailwindzie robić taką mieszankę?